daemon/containerd/image_list.go

```diff
@@ -154,9 +154,13 @@ func (i *ImageService) Images(ctx context.Context, opts imagetypes.ListOptions)
 
 	for _, img := range uniqueImages {
 		image, allChainsIDs, err := i.imageSummary(ctx, img, platformMatcher, opts, tagsByDigest)
-		if err != nil || image == nil {
+		if err != nil {
 			return nil, err
 		}
+		// No error, but image should be skipped.
+		if image == nil {
```

Comment:
I'm guessing that if `image` is not nil then `allChainsIDs` can't be nil?

Reply:
Correct